fiddler/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
```

Comment:
You can also add categories here with the following syntax:

`"Category::Alerting"`

For the list of possible categories, please see the word cloud at the top of [this page](https://docs.datadoghq.com/integrations/).